demo/dlb-init.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -eu
+
+enable_and_configure_vfs() {
+  devpath=$1
+  num_vfs_per_pf=$2
+  
+  sriov_numvfs_path="$devpath/sriov_numvfs"
+  if ! test -w "$sriov_numvfs_path"; then
+    echo "error: $sriov_numvfs_path is not found or not writable. Check if dlb driver module is loaded."
+    exit 1
+  fi
+  if [ "$(cat "$sriov_numvfs_path")" -ne 0 ]; then
+    echo "$devpath already configured"
+    exit 0
+  fi
+
+  #enable sriov
+  echo -n "$num_vfs_per_pf" > "$sriov_numvfs_path"
+
+  #configure vfs
```

Comment:
nit:
```suggestion
```

Reply:
removed, and added "assign resources to vf"!